__tests__/PlayingBoard_test.re

```diff
@@ -0,0 +1,102 @@
+open Jest;
+open Expect;
+open PlayingBoard;
+open SharedTypes;
+
+describe("module Game", () => {
+  describe("method createSandBoard", () =>
+    test("will create 3x2 playing board", () =>
+      createSandBoard((3, 2))
+      |> expect
+      |> toEqual([|[|Sand, Sand|], [|Sand, Sand|], [|Sand, Sand|]|])
+    )
+  );
+  describe("method randomizeBoard", () => {
+    test("will add Rocks to the board", () =>
+      randomizeBoard(
+        addRock,
+        3,
+        [|[|Sand, Sand|], [|Sand, Sand|], [|Sand, Sand|]|],
+      )
+      |> expect
+      |> toEqual([|[|Rock, Rock|], [|Sand, Sand|], [|Sand, Sand|]|])
```

Comment:
Try `toMatchSnapshot`, it should be easier in comparsion to explicit writing of expected structure.

Reply:
If it'll work, you can change rest of this randomizeBoard tests